scripts/ci/backport_audit/github_client.py

```
@@ -0,0 +1,116 @@
+        Returns:
+            List of event dictionaries
+
+        """
+        cmd = [
+            "gh", "api",
+            "--paginate",
+            f"repos/stackrox/stackrox/issues/{pr_number}/events",
+        ]
+
+        try:
+            result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)